src/platforms/javascript/common/session-replay/configuration.mdx

```diff
@@ -19,42 +19,45 @@ The following options can be configured on the root level of your browser-based
+| networkRequestHeaders  | string[]             | `[]`    | Capture request headers in addition to default headers.                                                                                                                         |
+| networkResponseHeaders | string[]             | `[]`    | Capture response headers in addition to default headers.                                                                                                                        |
 
-## Identifying Users
+### Sampling
 
-You can use the Sentry SDK to set the user of a session. To associate a user identity to a session replay, refer to the <PlatformLink to="/enriching-events/identify-user">Identifying User Docs</PlatformLink>.
+If you want to understand how exactly our session sampling works, check out our <PlatformLink to="/session-replay/understanding-sessions/#default-session-initialization">Understanding Sessions > Default Session Initialization</PlatformLink> documentation.
 
-```javascript
-Sentry.setUser({ email: "jane.doe@example.com" });
-```
+### Network Details
 
-## Start and Stop Recording
+By default, Replay will capture basic information about all outgoing fetch and XHR requests in your application. This includes the URL, request and response body size, method, and status code.
 
-Replay recording starts if it's included in the `integrations` array when calling `Sentry.init` or, when `addIntegration` is called from a Sentry client instance. To stop recording, call `stop()`:
+To capture additional information such as headers or bodies, you'll need to opt-in via `networkDetailAllowUrls`.
+Any URL matching the given pattern(s) will then be captured with additional details:
 
 ```javascript
-const replay = new Sentry.Replay();
-
-// Load replay when you configure Sentry SDK
-Sentry.init({
-  integrations: [replay],
+new Replay({
+   networkDetailAllowUrls: ['https://sentry.io/api']
 });
+```
 
-// Or if you want to defer loading Replay
-getCurrentHub()
-  .getClient()
-  .addIntegration(replay);
+If you give us a string, we'll match any URL that contains that string. You can use a Regex to handle exact or more complex matches.
 
-// sometime later
-await replay.flush(); // wait for any pending recording data to be sent
-replay.stop();
+Requests to a URL matched by the configured patterns will be enhanced with the request and response body, as well as the default headers `content-type`, `content-length`, and `accept, by default. If you want to capture additional headers, you'll have to configure them with the options `networkRequestHeaders` and `networkResponseHeaders`.
 
-replay.start();
+Note that captured bodies will be truncated to a maximum size of 150k characters. Content in bodies will be PII sanitized server-side, based on object keys and values.
```

Comment:
Maybe this isn't the best place to add this, but trying to tackle: https://github.com/getsentry/team-replay/issues/69


```suggestion
Note that captured bodies will be truncated to a maximum size of 150k characters. Content in bodies will be PII sanitized server-side, based on object keys and values.

### Sensitive data in request and response bodies

Sentry attempts to scrub certain types of sensitive data from request and response bodies, if you accidentally opt-in to a URL that includes sensitive data. This mechanism happens on our ingestion service, before data hits our disks. This is a best effort approach. Pattern matching the content with things like credit card information, social security numbers and [passwords](https://github.com/search?q=repo%3Agetsentry%2Frelay%20PASSWORD_KEY_REGEX&type=code). The best way to avoid PII getting into Sentry is to avoid adding the URL of endpoints that can contain PII.
```